_harp/eclipse_lyo/setup-an-oslc-provider-consumer-application.md

```diff
@@ -50,29 +68,86 @@ You should now have the project in Eclipse and the following folder structure:
 
 ![](./images/CreateMavenAdaptorProject_CodeStructure.png)
 
-## Customise the project POM file
+# Customise the project POM file
 
 We now need to modify the project *pom.xml* file.
 
-### General POM changes
+## Setup general POM properties
 
 We need to make sure our project uses UTF-8 and JDK 1.8. We will also use
-properties to define a common version for Lyo Core and Lyo Server packages:
+properties to define a common version for Lyo packages:
+
+Of course, ```ENTER-LYO-VERSION-HERE``` is either ```4.0.0-SNAPSHOT``` or ```2.4.0```, depending on your choice of JAX-RS version.
 
 ```xml
 <properties>
   <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
   <maven.compiler.source>1.8</maven.compiler.source>
   <maven.compiler.target>1.8</maven.compiler.target>
-  <version.lyo.core>2.2.0</version.lyo.core>
-  <version.lyo.server>2.2.0</version.lyo.server>
+  <version.lyo>ENTER-LYO-VERSION-HERE</version.lyo>
 </properties>
 ```
-### Java EE 6
+## Lyo repositories
+
+an OSLC4J project will use Lyo dependencies that we need to declare. Before we
+can do that, we need to add the Lyo repository. After that, we are ready to add the dependencies.
+
+```xml
+<repositories>
+  <repository>
+    <id>lyo-releases</id>
+    <name>Eclipse Lyo Releases</name>
+    <url>https://repo.eclipse.org/content/repositories/lyo-releases/</url>
+    <snapshots>
+      <enabled>false</enabled>
+    </snapshots>
+  </repository>
+  <repository>
+    <id>lyo-snapshots</id>
+    <name>Eclipse Lyo Snapshots</name>
+    <url>https://repo.eclipse.org/content/repositories/lyo-snapshots/</url>
+    <releases>
+      <enabled>false</enabled>
+    </releases>
+  </repository>
+</repositories>
+```
+
+## SLF4J package dependencies
+
+Lyo uses SLF4J for logging, leaving the choice of the actual logging library to
+use. We will use the simplest option:
+
+```xml
+<dependency>
+  <groupId>org.slf4j</groupId>
+  <artifactId>slf4j-simple</artifactId>
+  <version>1.7.21</version>
+  <scope>runtime</scope>
+</dependency>
+```
+
+## Servlet dependencies
 
 We require Java EE 6 or higher and JSTL:
 
+### for Lyo 4.0.0-SNAPSHOT 
```

Comment:
Why is there a difference?

Reply:
you tell me :-) I'm happy to keep it as is, until we work out the difference.